devbox.go

```diff
@@ -32,18 +32,18 @@ type Devbox interface {
 	Install(ctx context.Context) error
 	IsEnvEnabled() bool
 	ListScripts() []string
-	PrintEnv(ctx context.Context, includeHooks bool) (string, error)
+	PrintEnv(ctx context.Context, includeHooks bool, pure bool) (string, error)
```

Comment:
Not a blocker

I understand that adding boolean is the most straight forward way to do this, but boolean params are generally code smell and make code harder to read. Imagine `doStuff(true, false, true)` you need to add comments or check signature every time.

A few alternatives:

* Use named types. `PrintEnv(ctx, false, pure)`
* Use options struct `PrintEnv(ctx, opts{Pure: true})`